src/main/java/uk/gov/hmcts/reform/bulkscan/payment/processor/service/servicebus/model/PaymentInfo.java

```diff
@@ -0,0 +1,17 @@
+package uk.gov.hmcts.reform.bulkscan.payment.processor.service.servicebus.model;
+
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+@JsonIgnoreProperties(ignoreUnknown = true)
+public class PaymentInfo {
+
+    public final String documentControlNumbers;
+
+    public PaymentInfo(@JsonProperty(value = "document_control_numbers", required = true)
```

Comment:
I think the property name should be `document_control_number`?

Reply:
fixed